plugins/src/test/java/com/google/fhir/gateway/plugin/ListAccessCheckerTest.java

```diff
@@ -247,6 +247,14 @@ public void canAccessBundlePostPatient() throws IOException {
     assertThat(testInstance.checkAccess(requestMock).canAccess(), equalTo(true));
   }
 
+  @Test
+  public void canAccessBundleDeletePatient() throws IOException {
+    // Query: POST / -d @bundle_transaction_delete_patient.json
+    setUpFhirBundle("bundle_transaction_delete_patient.json");
+    AccessChecker testInstance = getInstance();
+    assertThat(testInstance.checkAccess(requestMock).canAccess(), equalTo(true));
```

Comment:
Do we have a test-case where a Patient that is not in user's access-list is deleted (so it is denied)? If not, can you please add one?